app/static/js/solarsense.js

```diff
@@ -46,21 +46,15 @@ app.controller('HomeCtrl', function($scope, $timeout, $http, $window) {
 	})
 
 	$scope.notifications = [];
-	// Temporary Variables, until we properly pull from databases
 	$scope.statusWarnings = {
 		"OK" : 0,
 		"Caution" : 1,
 		"Warning" : 2
 	}
 	$scope.cropName = "";
-	$scope.temperatureStatus = 0; // OK (Green)
-	$scope.sunlightStatus = 2; // Caution (Yellow)
-	$scope.waterStatus = 0; // Warning (Red)
-
-	// Debug Values (Most likely not to be used in final product)
-	// $scope.temperature = 70.0;
-	// $scope.sunlightTime = 4.0;
-	// $scope.waterAmount = 10000.0;
+	$scope.temperatureStatus = 1;
```

Comment:
We could make this 0 1 2 so that we can make sure that each statusWarning is working